sdk/test-utils/perf/runAllTests.sh

```diff
@@ -3,11 +3,8 @@
 rushx clean
 rushx build
 
+npm run perf-test:node -- MockReceiverTest --warmup 2 --duration 4 --iterations 2
 npm run perf-test:node -- NoOp --warmup 2 --duration 4 --iterations 2
-npm run perf-test:node -- NoOp --warmup 2 --duration 4 --iterations 2 --sync
-npm run perf-test:node -- OptionsTest --warmup 2 --duration 4 --iterations 2 --sync
-npm run perf-test:node -- SetupCleanupTest --warmup 2 --duration 4 --iterations 2 --sync
 npm run perf-test:node -- Delay500ms --warmup 2 --duration 4 --iterations 2
```

Comment:
Is this intentional?

Reply:
We removed sync in an older PR.
All we now test is async since that's what we provide in our SDKs.

Reply:
ok ok!